docs/user-guide/grpc.md

```diff
@@ -0,0 +1,136 @@
+# gRPC example
+
+This section explains how to use Traefik has reverse proxy for gRPC application with self-signed certificates.
+
+!!! warning
+    Has gRPC needs HTTP2, we need to have valid HTTPS on both gRPC Server and Traefik.
+
+<p align="center">
+<img src="/img/grpc.svg" alt="gRPC architecture" title="gRPC architecture" />
+</p>
+
+## gRPC Server certificate
+In order to secure the gRPC server, we generate a self-signed certificate for backend url.
+```bash
+ openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout ./backend.key -out ./backend.crt
+```
+
+That will prompt for information, the important answer is
+```
+Common Name (e.g. server FQDN or YOUR name) []: backend.local
+```
+
+## gRPC Client certificate
+Generate your self-signed certificate for frontend url
+```bash
+openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout ./frontend.key -out ./frontend.crt
+```
+with
+```
+Common Name (e.g. server FQDN or YOUR name) []: frontend.local
+```
+
+## Traefik configuration
+
+At last, we configure our Traefik to use the both self-signed certificate.
+
+```toml
+defaultEntryPoints = ["https"]
+
+# For secure connection on backend.local
+RootCAs = [ "./backend.cert" ]
+
+[entryPoints]
+  [entryPoints.https]
+  address = ":4443"
+    [entryPoints.https.tls]
+     # For secure connection on frontend.local
+     [[entryPoints.https.tls.certificates]]
+     CertFile = "./frontend.cert"
+     KeyFile  = "./frontend.key"
+
+
+[web]
+  address = ":8080"
+
+[file]
+
+[backends]
+  [backends.backend1]
+    [backends.backend1.servers.server1]
+    # Access on backend with HTTPS
+    url = "https://backend.local:8080"
+
+
+[frontends]
+  [frontends.frontend1]
+  backend = "backend1"
+    [frontends.frontend1.routes.test_1]
+    rule = "Host:frontend.local"
+```
+
+## Conclusion
+
+We don't need specific configuration to use gRPC in Traefik, we just need to be careful that all the exchange (between client and Traefik, and between Traefik and backend) are valid HTTPS communication (no InsecureSkipVerify) because gRPC use HTTP2.
+
```

Comment:
s/Traefik/Træfik/
s/exchange/exchange*s*/ ?
s/communication/communication*s*/ ?

> no InsecureSkipVerify

=> _without the `skipInsecureVerify` option enabled_